.github/workflows/test.yaml

```diff
@@ -14,6 +14,7 @@ jobs:
       # Exclude features not supported by rbpf yet.
       # atomic operations - https://github.com/qmonnet/rbpf/issues/47
       # mem-len - Tests assume r2 contains the length of the memory.
+      RUSTFLAGS: "-C instrument-coverage"
```

Comment:
`RUSTFLAGS` is not used in the commands below, can you please confirm whether it's still necessary? If it is passed as an environment variable to one of the commands, could we please add a comment to specify it?